LondonBritishLibrary/orient/BLorient8822.xml

```diff
@@ -0,0 +1,162 @@
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1+88+2</measure>
+                                    <measure unit="leaf" type="blank" corresp="#1v #III">2</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>240</height>
+                                        <width>190</width>
+                                    </dimensions>
+                                </extent></supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="27">
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Rather careful handwriting.</desc>
+                                <date notBefore="1700" notAfter="1799" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="OwnershipNote">Note inside front cover.</desc>
+                                    <q xml:lang="en"><persName ref="PRS14319Curzon">R. CVRZON</persName></q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#Ir"/>
+                                    <desc type="AcquisitionNote"/>
+                                    <q xml:lang="en">Bequeathed by <persName ref="PRS14320DareaCurzon">Darea Baroness Zouche</persName>. 
+                                        <date when="1917">13 Oct. 1917</date>.</q>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#Iv #1r #II"/>
+                                    <desc>Pen trials, scribbles.</desc>  
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1"/>
+                                <decoNote type="bindingMaterial" xml:id="b2"><material key="wood"/></decoNote>
+                                <decoNote xml:id="b3" type="Spine"><material key="leather">The leather spine added in rebinding in 
+                                    <placeName ref="INS00001BL"/>.</material></decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
```

Comment:
I think it would be useful to add that the manuscript was acquired by Curzon in Egypt (I send you a recent article on his collection by email)